plugins/out_s3/s3.c

```diff
@@ -1627,8 +1819,8 @@ static struct flb_config_map config_map[] = {
     "by the rewrite_tag filter. Add $TAG in the format string to insert the full "
     "log tag; add $TAG[0] to insert the first part of the tag in the s3 key. "
     "The tag is split into “parts” using the characters specified with the "
-    "s3_key_format_tag_delimiters option. See the in depth examples and tutorial"
-    " in the documentation."
+    "s3_key_format_tag_delimiters option. Add $INDEX to enable sequential indexing "
+    "for file names. See the in depth examples and tutorial in the documentation."
```

Comment:
We might need to add one part here that if $INDEX is supported, we will not have random string here.